### PR TITLE
add new option after drinking a coffee

### PR DESCRIPTION
**Pull request by jmitchell-au**
Please review this, @casey-collab

### CODE REVIEW

File: english/coffee/coffee.md [hunk 1]
```diff
@@ -9,3 +9,5 @@ After having your coffee do you want to:
 [Write a caffeine-fueled full-length novel?](novel/full-length-novel.md)
 
 [Go skinny dipping?](nude-run/nude-run.md)
+
+[Light a Fire?](light-fire/fire.md)
```
Comment (casey-collab) [inline english/coffee/coffee.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :dog:

**Pull request merged**